### ISSUE TITLE
whitespace/layout changes to python.Portfile template

### ISSUE THREAD

**Issue by reneeotten** [macports/upt-macports#5]
Ideally, the layout/whitespaces within the ```if {${name} ne ${subport}} { }``` block should be the same irrespective of which/how many dependency types are present.

Try packaging ```setuptools```, ```upt``` and ```pytest``` as examples of the different scenarios and make sure the layout is the same as below (i.e., no newline at the beginning of the if-block and one newline between the different dependency types, and the livecheck):
```
if {${name} ne ${subport}} {
    depends_build-append \
                    port:py${python.version}-setuptools

    depends_lib-append \
                    port:py${python.version}-???

    depends_test-append \
                    port:py${python.version}-???

   livecheck.type  none
```

**Comment by Steap** [comment]
Is there a linter that could help us check that the spacing is right?

**Comment by reneeotten** [comment]
> Is there a linter that could help us check that the spacing is right?

no, there is not. ```port lint --nitpick <portname>``` will detect several issues, but regardless of the whitespace the syntax of the Portfile is correct. It's just that for most Portfiles this whitespacing is used and (in my opinion) it also adds to the readability so I would like to have ```upt``` generate it like this.

**Comment by Steap** [comment]
Fixed in  156b5ed.
